Cannot build Cordova Android with latest 2.10.0 version

**Description:**
<!-- (write below this line) -->

Cannot build Cordova Android with latest 2.10.0 version

Get the following errors
/platforms/android/gradlew: Command failed with exit code 1 Error output:
platforms/android/app/src/main/java/com/plugin/gcm/OneSignalOutcomeController.java:25: error: unreported exception JSONException; must be caught or declared to be thrown
            CallbackHelper.callbackSuccess(jsSendUniqueOutcomeCallback, outcomeEvent.toJSONObject());
                                                                                                 ^
/platforms/android/app/src/main/java/com/plugin/gcm/OneSignalOutcomeController.java:45: error: unreported exception JSONException; must be caught or declared to be thrown
            CallbackHelper.callbackSuccess(jsSendOutcomeCallback, outcomeEvent.toJSONObject());
                                                                                           ^
/platforms/android/app/src/main/java/com/plugin/gcm/OneSignalOutcomeController.java:66: error: unreported exception JSONException; must be caught or declared to be thrown
            CallbackHelper.callbackSuccess(jsSendOutcomeWithValueCallback, outcomeEvent.toJSONObject()); 

**Environment**
1. What version of the Cordova SDK are you using? 2.10.0
2. Provide a list of your project dependencies (using `cordova plugin list` or copy-pasting your config.xml)

Ionic Info
Ionic:

   Ionic CLI         : 5.2.3 (/Users/jasonandress/.nvm/versions/node/v12.6.0/lib/node_modules/ionic)
   Ionic Framework   : ionic1 1.3.4
   @ionic/v1-toolkit : 2.0.6

Cordova:

   Cordova CLI       : 9.0.0 (cordova-lib@9.0.1)
   Cordova Platforms : android 8.1.0, ios 6.0.0
   Cordova Plugins   : cordova-plugin-ionic-keyboard 2.2.0, cordova-plugin-ionic-webview 5.0.0, (and 23 other plugins)

Utility:

   cordova-res : 0.6.0 
   native-run  : 0.2.8 

System:

   Android SDK Tools : 26.1.1 (/Users//Library/Android/sdk)
   ios-deploy        : 1.9.4
   ios-sim           : 8.0.2
   NodeJS            : v12.6.0 (/Users//.nvm/versions/node/v12.6.0/bin/node)
   npm               : 6.9.0
   OS                : macOS Catalina
   Xcode             : Xcode 11.5 Build version 11E608c


cordova-admob-plus 0.0.0 "@admob-plus/cordova"
cordova-annotated-plugin-android 1.0.4 "cordova-annotated-plugin-android"
cordova-plugin-add-swift-support 2.0.2 "AddSwiftSupport"
cordova-plugin-androidx 2.0.0 "cordova-plugin-androidx"
cordova-plugin-androidx-adapter 1.1.1 "cordova-plugin-androidx-adapter"
cordova-plugin-calendar 5.1.5 "Calendar"
cordova-plugin-camera 4.1.0 "Camera"
cordova-plugin-device 2.0.2 "Device"
cordova-plugin-dialogs 1.3.0 "Notification"
cordova-plugin-enable-multidex 0.2.0 "Enable Multidex"
cordova-plugin-facebook4 6.4.0 "Facebook Connect"
cordova-plugin-firebase-analytics 4.3.0 "FirebaseAnalyticsPlugin"
cordova-plugin-firebase-crash 2.2.0 "cordova-plugin-firebase-crash"
cordova-plugin-inappbrowser 4.0.0 "InAppBrowser"
cordova-plugin-intercom 8.0.0 "Intercom"
cordova-plugin-ionic-keyboard 2.2.0 "cordova-plugin-ionic-keyboard"
cordova-plugin-ionic-webview 5.0.0 "cordova-plugin-ionic-webview"
cordova-plugin-network-information 2.0.2 "Network Information"
cordova-plugin-purchases 1.1.0 "Purchases"
cordova-plugin-settings-hook 0.2.7 "Cordova Settings Hooks"
cordova-plugin-splashscreen 5.0.4 "Splashscreen"
cordova-plugin-statusbar 2.4.3 "StatusBar"
cordova-plugin-whitelist 1.3.3 "Whitelist"
cordova-support-android-plugin 1.0.2 "cordova-support-android-plugin"
cordova-support-google-services 1.4.0 "cordova-support-google-services"
onesignal-cordova-plugin 2.9.1 "OneSignal Push Notifications"


**Steps to Reproduce Issue:**

1. Add 2.10.0 plugin version
2. Build android -- cordova run android
3. get errors above
4. Install 2.9.1 - compiles and runs with no errors

**Anything else:**

(crash stacktraces, as well as any other information here)


<!--
  SEARCH EXISTING ISSUES FOR AN ANSWER: https://goo.gl/G4XxN2
-->

--- a/src/android/com/plugin/gcm/OneSignalOutcomeController.java
+++ b/src/android/com/plugin/gcm/OneSignalOutcomeController.java
@@ -1,5 +1,7 @@
 package com.plugin.gcm;
 
+import android.util.Log;
+
 import com.onesignal.OneSignal;
 import com.onesignal.OutcomeEvent;
 import com.onesignal.OneSignal.OutcomeCallback;
@@ -12,17 +14,24 @@ import org.json.JSONObject;
 
 public class OneSignalOutcomeController {
 
+  private static final String TAG = "OneSignalOutcome";
+
   public static boolean sendUniqueOutcome(CallbackContext callbackContext, JSONArray data) {
     try {
       final CallbackContext jsSendUniqueOutcomeCallback = callbackContext;
-      String name = data.getString(0);
+      final String name = data.getString(0);
       OneSignal.sendUniqueOutcome(name, new OutcomeCallback(){
         @Override
         public void onSuccess(OutcomeEvent outcomeEvent) {
           if (outcomeEvent == null)
             CallbackHelper.callbackSuccess(jsSendUniqueOutcomeCallback, new JSONObject());
-          else
-            CallbackHelper.callbackSuccess(jsSendUniqueOutcomeCallback, outcomeEvent.toJSONObject());
+          else {
+            try {
+              CallbackHelper.callbackSuccess(jsSendUniqueOutcomeCallback, outcomeEvent.toJSONObject());
+            } catch (JSONException e) {
+              Log.e(TAG, "sendUniqueOutcome with name: " + name + ", failed with message: " + e.getMessage());
+            }
+          }
         }
       });
       return true;
@@ -35,14 +44,19 @@ public class OneSignalOutcomeController {
   public static boolean sendOutcome(CallbackContext callbackContext, JSONArray data) {
     try {
       final CallbackContext jsSendOutcomeCallback = callbackContext;
-      String name = data.getString(0);
+      final String name = data.getString(0);
       OneSignal.sendOutcome(name, new OutcomeCallback() {
         @Override
         public void onSuccess(OutcomeEvent outcomeEvent) {
           if (outcomeEvent == null)
             CallbackHelper.callbackSuccess(jsSendOutcomeCallback, new JSONObject());
-          else
-            CallbackHelper.callbackSuccess(jsSendOutcomeCallback, outcomeEvent.toJSONObject());
+          else {
+            try {
+              CallbackHelper.callbackSuccess(jsSendOutcomeCallback, outcomeEvent.toJSONObject());
+            } catch (JSONException e) {
+              Log.e(TAG, "sendOutcome with name: " + name + ", failed with message: " + e.getMessage());
+            }
+          }
         }
       });
       return true;
@@ -55,15 +69,20 @@ public class OneSignalOutcomeController {
   public static boolean sendOutcomeWithValue(CallbackContext callbackContext, JSONArray data) {
     try {
       final CallbackContext jsSendOutcomeWithValueCallback = callbackContext;
-      String name = data.getString(0);
-      float value = Double.valueOf(data.optDouble(1)).floatValue();
+      final String name = data.getString(0);
+      final float value = Double.valueOf(data.optDouble(1)).floatValue();
       OneSignal.sendOutcomeWithValue(name, value, new OutcomeCallback() {
         @Override
         public void onSuccess(OutcomeEvent outcomeEvent) {
           if (outcomeEvent == null)
             CallbackHelper.callbackSuccess(jsSendOutcomeWithValueCallback, new JSONObject());
-          else
-            CallbackHelper.callbackSuccess(jsSendOutcomeWithValueCallback, outcomeEvent.toJSONObject());
+          else {
+            try {
+              CallbackHelper.callbackSuccess(jsSendOutcomeWithValueCallback, outcomeEvent.toJSONObject());
+            } catch (JSONException e) {
+              Log.e(TAG, "sendOutcomeWithValue with name: " + name + " and value: " + value + ", failed with message: " + e.getMessage());
+            }
+          }
         }
       });
       return true;

